proto/uber/cadence/api/v1/domain.proto

```diff
@@ -63,6 +63,13 @@ message BadBinaryInfo {
   google.protobuf.Timestamp created_time = 3;
 }
 
+message FailoverInfo {
+  int64 failover_version = 1;
+  int64 failover_expired_timestamp = 2;
```

Comment:
is it possible to have failover_started_timestamp here?

Reply:
We don't store this in DB. But I can calculate the start time based on the expired time. What do you think?

Reply:
I think it's a useful information especially in case we are going to show this in the UI as well. If it's going to be a too much work, we can talk about postponing this to a later time.

Reply:
Yes. Added the start timestamp